src/components/TodoList.js

```diff
@@ -8,7 +8,7 @@ function TodoList({ todoList, onRemoveTodo }) {
       {todoList.map((item) => (
         <TodoListItem
           key={item.id}
-          id={item.id}
+          id={parseInt(item.id)}
```

Comment:
I must have missed it in your changes to the last assignment but for some reason you have `id: PropTypes.number.isRequired` in `TodoListItem.js` when it should be of type string, making this parsing unnecessary.